data/mods/Magiclysm/worldgen/forest_tomb.json

```diff
@@ -0,0 +1,138 @@
+[
+  {
+    "type": "palette",
+    "id": "tomb",
+    "terrain": {
+      ".": [ "t_rock_floor" ],
```

Comment:
can remove the [ ] for all of these except ones that have an array of terrains. Format will be:
`".": "t_rock_floor",`